src/Faker/Provider/nl_NL/Person.php

```diff
@@ -346,4 +377,46 @@ public function idNumber()
         }
         return implode('', array_reverse($nr));
     }
+
+    /**
+     * @param \DateTime   $issueDate
+     * @param string|null $documentCode   Key from $idStartLetter
+     *
+     * @return string
+     */
+    public function identityDocumentNumber(
+        \DateTime $issueDate,
+        string $documentCode = null
+    ) {
+        if (!isset($documentCode)) {
+            $documentCode = static::randomKey(static::$idStartLetter);
+        }
+        $startLetter = static::$idStartLetter[$documentCode];
+
+        // No '0' in documents issued from 1 december 2019
+        $extra = [];
+        if ($issueDate < new \DateTime('2020-12-01')) {
+            array_push($extra, 0);
+        }
+
+        return implode(
+            '',
+            array_merge(
+                [$startLetter],
+                [static::randomElement(static::$idLetters)],
+                static::randomElements(array_merge(static::$idLetters, static::$idNumbers, $extra), 7, true),
+                [static::randomElement(static::$idNumbers)],
```

Comment:
PHP <7.3 doesn't like trailing commas in function calls - please remove the trailing comma.

Reply:
Thanks!